src/actions/__tests__/config.spec.js

```diff
@@ -1,117 +1,120 @@
+import { fromJS } from 'immutable';
```

Comment:
Causes for change in this file:

* Config objects are now converted to immutable maps before processing, so functions like `applyDefaults` now expect a map.